build/database/01.Database_Schema.sql

```diff
@@ -1,22 +1,35 @@
      CREATE TABLE IF NOT EXISTS membership_incentive (membership_id INT NOT NULL, incentive_id INT NOT NULL, INDEX IDX_4AE7CF6F1FB354CD (membership_id), INDEX IDX_4AE7CF6FF17F400F (incentive_id), PRIMARY KEY(membership_id, incentive_id)) DEFAULT CHARACTER SET utf8 COLLATE `utf8_unicode_ci` ENGINE = InnoDB;
      CREATE TABLE IF NOT EXISTS incentive (id INT AUTO_INCREMENT NOT NULL, name VARCHAR(32) NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE `utf8_unicode_ci` ENGINE = InnoDB;
+     CREATE TABLE IF NOT EXISTS payments_bank_transfer (id INT NOT NULL, payment_reference_code VARCHAR(255) DEFAULT NULL, is_cancelled TINYINT(1) NOT NULL, UNIQUE INDEX UNIQ_F63CD417E5FE723C (payment_reference_code), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE `utf8_unicode_ci` ENGINE = InnoDB;
+     CREATE TABLE IF NOT EXISTS payments_credit_card (id INT NOT NULL, valuation_date DATETIME DEFAULT NULL COMMENT '(DC2Type:datetime_immutable)', booking_data LONGTEXT DEFAULT NULL COMMENT '(DC2Type:json)', PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE `utf8_unicode_ci` ENGINE = InnoDB;
+     CREATE TABLE IF NOT EXISTS payments_direct_debit (id INT NOT NULL, iban VARCHAR(255) DEFAULT NULL, bic VARCHAR(255) DEFAULT NULL, is_cancelled TINYINT(1) NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE `utf8_unicode_ci` ENGINE = InnoDB;
+     CREATE TABLE IF NOT EXISTS payments_paypal (id INT NOT NULL, parent_payment_id INT DEFAULT NULL, valuation_date DATETIME DEFAULT NULL COMMENT '(DC2Type:datetime_immutable)', booking_data LONGTEXT DEFAULT NULL COMMENT '(DC2Type:json)', INDEX IDX_2A359C3C438027EB (parent_payment_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE `utf8_unicode_ci` ENGINE = InnoDB;

```

Comment:
This is missing the new `transaction_id` field and index from https://github.com/wmde/fundraising-payments/pull/109 , the missing fields might fail the booking tests